### PR TITLE
Script to populate address tables

### PR DESCRIPTION
**Pull request by pdinarte**
### Changelog
- Created sql script to insert data into address tables
- CSV file with data. ([link to original](https://guiascostarica.com/division-administrativa-costa-rica/))

### Ticket 
https://github.com/akurey/animal-rescue-backend/issues/46

### Solution
- Created a temporary table to insert the data from the csv. That table is used to insert the values into Provinces, Cantons and Districts.

### Considerations
The script uses an insert statement with copy-pasted values from the csv. Data could also be copied directly from the csv running the following command:
`\copy temp_table(province, canton, district) FROM '_path_goes_here_\zonas_cr.csv' DELIMITER ',' CSV HEADER;`
However, accent marks in the rows of the csv were not encoded properly therefore the committed solution was the best option.

### CODE REVIEW

File: database/zonas_cr.csv [hunk 1]
```diff
@@ -0,0 +1,488 @@
+Provincia,Cantón,Distrito
```
Comment (rnavarroSF) [inline database/zonas_cr.csv:1]: What value did you see in storing this csv?

File: database/010 - populate_address_tables.sql [hunk 2]
```diff
@@ -0,0 +1,511 @@
...
+SELECT canton, P.id FROM "temp_zonas"
+INNER JOIN "AP_Provinces" P ON P.name = "temp_zonas".province
+GROUP BY canton, P.id
+ORDER BY P.id, canton;
+
+INSERT INTO "AP_Districts" (name, canton_id)
+SELECT district, C.id FROM "temp_zonas"
+INNER JOIN "AP_Cantons" C ON C.name = "temp_zonas".canton
+GROUP BY district, C.id
+ORDER BY C.id, district;
```
Comment (rnavarroSF) [inline database/010 - populate_address_tables.sql:511]: Can you delete 'temp_zonas' after this?

**Review by rnavarroSF — CHANGES_REQUESTED** [review]

File: database/zonas_cr.csv (hunk 1 above)
Comment (pdinarte) [reply database/zonas_cr.csv:1]: Sorry, I forgot to delete it. I was going to populate tables from there but at the end I didn't do it that way so is not necessary anymore.

**Review by rnavarroSF — APPROVED** [review]
No worries! 🚀

**Pull request merged**